src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -107,11 +151,12 @@ export class CreateSnapshotModal extends React.Component {
         // cut off seconds, subseconds, and timezone
         const now = new Date().toISOString()
                 .replace(/:[^:]*$/, '');
-        const snapName = props.vm.name + '_' + now;
+        const snapName = now;
```

Comment:
The commit message mentions the "what": "The default snapshot name does not include the VM name", but not the "why". Is the VM name already included in the path anyway, also for disk snapshots? I find it quite important to have it in the *files*, so that admins can use shell tools to associate them to VMs. Of course the UI doesn't need to repeat the VM name in the snapshot name. 

Update to myself: It gets set in `mpath` in line 216 below.

Reply:
> It gets set in mpath in line 216 below.

Yes, exactly.